lib/senders/email_service.js

```diff
@@ -5,6 +5,14 @@
 'use strict'
 
 const request = require('request')
+const error = require('../error')
+
+const ERRNO = {
+  // From fxa-email-service, src/app_errors/mod.rs
+  COMPLAINT: '106',
+  SOFT_BOUNCE: '107',
+  HARD_BOUNCE: '108'
```

Comment:
I was surprised, but these are strings in the `body` object we get back from `request`.